modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/either.kt

```diff
@@ -149,16 +152,34 @@ interface EitherFoldable<L> : Foldable<EitherPartialOf<L>> {
     fix().foldRight(lb, f)
 }
 
+@extension
+interface EitherBifoldable : Bifoldable<ForEither> {
+  override fun <A, B, C> EitherOf<A, B>.bifoldLeft(c: C, f: (C, A) -> C, g: (C, B) -> C): C =
+    fix().fold({ f(c, it) }, { g(c, it) })
+
+  override fun <A, B, C> EitherOf<A, B>.bifoldRight(c: Eval<C>, f: (A, Eval<C>) -> Eval<C>, g: (B, Eval<C>) -> Eval<C>): Eval<C> =
+    fix().fold({ f(it, c) }, { g(it, c) })
+}
+
 fun <G, A, B, C> EitherOf<A, B>.traverse(GA: Applicative<G>, f: (B) -> Kind<G, C>): Kind<G, Either<A, C>> =
   fix().fold({ GA.just(Either.Left(it)) }, { GA.run { f(it).map { Either.Right(it) } } })
 
+fun <G, A, B, C, D> EitherOf<A, B>.bitraverse(AP: Applicative<G>, f: (A) -> Kind<G, C>, g: (B) -> Kind<G, D>):
```

Comment:
This function is unnecessary at this level, even if it's possible to implement.

Reply:
Do you mean to move inside of EitherBitraverse?

Reply:
Yes. Bifoldable will mostly be used as an interface, so it's not expected for this to be here. There's only some cases where we've done it in Composed instances.

Reply:
yes, I totally agree with you